configuration-example.js

```diff
@@ -1,3 +1,12 @@
 const EluvioConfiguration = {
-  "config-url": "https://main.net955304.contentfabric.io/config"
-};
+  // Fabric Network
```

Comment:
this and the readme need details on
`fabricBrowserApps`,
`displayAppUrl`,
`embedAppUrl`

see https://core.v3.contentfabric.io/configuration.js for defaults

Reply:
Please also specify that `coreUrl` should be changed to `http://localhost:8082` (local core url).